sopel/config/core_section.py

```diff
@@ -216,3 +216,9 @@ def homedir(self):
 
     verify_ssl = ValidatedAttribute('verify_ssl', bool, default=True)
     """Whether to require a trusted SSL certificate for SSL connections."""
+
+    auto_url_schemes = ListAttribute(
+        'auto_url_schemes',
+        strip=True,
+        default=('http', 'https', 'ftp'))
+    """List of URL schemes the URL callbacks are triggered."""
```

Comment:
```suggestion
    """List of URL schemes that will trigger URL callbacks."""
```

Need to think about whether it's possible, but this should link to documentation for that API feature. Maybe `bot.register_url_callback()`?